components/bodyview.gd

```diff
@@ -17,6 +20,20 @@ func _ready():
 	sprite.set_texture(load("res://assets/bodies/" + body.type + "/idle.tex"))
 	set_process(true)
 
+func highlight():
+	highlight = true
+	update_hl()
+
+func unhighlight():
+	highlight = false
+	update_hl()
+
+func update_hl():
+	if highlight:
+		get_node("Highlight").show()
+	else:
+		get_node("Highlight").hide()
```

Comment:
No,  it seems that variable is only ready after the unserialize process. Then during the load saved file it is null.